.pre-commit-config.yaml

```diff
@@ -0,0 +1,5 @@
+repos:
+- repo: https://github.com/BlankSpruce/gersemi
+  rev: 0.16.2
+  hooks:
+  - id: gersemi
```

Comment:
Where does this get executed? This won't configure my local git repository, right?

Reply:
This will run on local machine if you enable it.

It would get enabled if you do
```
pre-commit install
(modify a CMakeLists.txt file)
git add that_file
pre-commit run
```

I think it can also run when you create the commit.
If you don't want the changes it suggests, you can do git commit --no-verify

pre-commit hooks are becoming quite standard in most repos these days.